cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaBridgeCluster.java

```diff
@@ -403,6 +404,7 @@ protected List<EnvVar> getEnvVars() {
             varList.add(ContainerUtils.createEnvVar(ENV_VAR_KAFKA_BRIDGE_TRUSTED_CERTS, CertUtils.trustedCertsEnvVar(tls.getTrustedCertificates())));
         }
 
+        // This is still needed to generate oauth truststore certificates in PKCS12 format
```

Comment:
This comment makes sense in the context of this PR, but I wonder if it will be confusing when viewed in isolation in future. What about:
```suggestion
        // Client authentication env var is needed to generate oauth truststore certificates in PKCS12 format in container script
```